Resources/config/services.yml

```diff
@@ -26,6 +26,17 @@ services:
         tags:
             - { name: 'enqueue.client.processor'}
 
+    service.event.processor:
+        class: Garlic\Bus\Service\Processor\MulticastEventProcessor
+        public: true
+        arguments:
+            - "@service.request"
+            - "@service.response"
+            - "@router"
+            - "@kernel"
+        tags:
+            - { name: 'enqueue.client.processor', topicName: 'multicast_event'}
```

Comment:
Can you exclude topicName to parrameter and add Namespace to them?

Reply:
It's hardcoded value for inner usage, no need to fetch it into separate place.

Reply:
fixed